scripts/drone/steps/lib.star

```diff
@@ -181,27 +181,38 @@ def lint_drone_step():
         ],
     }
 
-
-def enterprise_downstream_step(edition):
+def enterprise_downstream_step(edition, ver_mode):
     if edition in ('enterprise', 'enterprise2'):
         return None
 
-    return {
+    repo = 'grafana/grafana-enterprise@'
+    if ver_mode == 'pr':
+        repo += '${DRONE_SOURCE_BRANCH}'
+    else:
+        repo += 'main'
+
+    step = {
         'name': 'trigger-enterprise-downstream',
         'image': 'grafana/drone-downstream',
         'settings': {
             'server': 'https://drone.grafana.net',
             'token': from_secret(drone_token),
             'repositories': [
-                'grafana/grafana-enterprise@main',
+                repo,
             ],
             'params': [
                 'SOURCE_BUILD_NUMBER=${DRONE_COMMIT}',
                 'SOURCE_COMMIT=${DRONE_COMMIT}',
+                'OSS_PULL_REQUEST=${DRONE_PULL_REQUEST}',
             ],
         },
     }
 
+    if ver_mode == 'pr':
+        step.update({ 'failure': 'ignore' })
+
```

Comment:
That's only for testing purposes I assume? When we are sure that this is working we are going to remove it?

Reply:
No it's because this step is supposed to fail when the Enterprise branch doesn't exist.